src/utils/actions.js

```diff
@@ -230,6 +230,77 @@ const actions = [
     },
     requiredInput: ['partial'],
   },
+  {
```

Comment:
This action will need to be behind a feature flag. On QSEfW this endpoint will not exist and it should be disabled.